modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/buttons/TimePlayButton.kt

```diff
@@ -0,0 +1,138 @@
+package au.com.shiftyjelly.pocketcasts.compose.buttons
+
+import androidx.annotation.StringRes
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.OutlinedButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.clearAndSetSemantics
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import au.com.shiftyjelly.pocketcasts.compose.AppThemeWithBackground
+import au.com.shiftyjelly.pocketcasts.compose.components.TextH40
+import au.com.shiftyjelly.pocketcasts.compose.theme
+import au.com.shiftyjelly.pocketcasts.localization.R
+import au.com.shiftyjelly.pocketcasts.localization.helper.TimeHelper
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
+import com.airbnb.android.showkase.annotation.ShowkaseComposable
+
+sealed class TimePlayButtonStyle {
+    object Solid : TimePlayButtonStyle()
+    object Outline : TimePlayButtonStyle()
+}
+
+@Composable
+fun TimePlayButton(
+    timeSecs: Int,
+    @StringRes contentDescriptionId: Int,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    textColor: Color = MaterialTheme.theme.colors.primaryText01,
+    backgroundColor: Color = MaterialTheme.colors.surface,
+    borderColor: Color = MaterialTheme.theme.colors.primaryText01,
+    buttonStyle: TimePlayButtonStyle = TimePlayButtonStyle.Outline
+) {
+    val timeText by remember {
+        mutableStateOf(TimeHelper.formattedSeconds(timeSecs.toDouble()))
+    }
+    val description = stringResource(contentDescriptionId, timeText)
+    val border = when (buttonStyle) {
+        TimePlayButtonStyle.Outline -> BorderStroke(2.dp, borderColor)
+        TimePlayButtonStyle.Solid -> null
+    }
+    OutlinedButton(
```

Comment:
I noticed the highlight was escaping the button shape so I have switched it to an OutlinedButton.
<img width="167" alt="Screenshot 2023-07-22 at 10 06 51 am" src="https://github.com/Automattic/pocket-casts-android/assets/308331/4c00952a-08d4-4608-9e5b-7d2cea614fd6">